stake-pool/program/src/processor.rs

```diff
@@ -582,7 +601,7 @@ impl Processor {
         stake_pool.pool_total += token_amount;
```

Comment:
I can't comment further up, but you'll need to add the activated check a little higher up at the TODO as well